test/test_util.go

```diff
@@ -92,3 +105,11 @@ type ValidationFunctionFailed struct {
 func (err ValidationFunctionFailed) Error() string {
 	return fmt.Sprintf("Validation failed for URL %s. Response status: %d. Response body:\n%s", err.Url, err.Status, err.Body)
 }
+
+func TestWebsite(t *testing.T, protocol string, url string, path string, expectedStatus int, expectedBody string) {
```

Comment:
Will this function name confuse `go test` because it also starts with `Test`?

Reply:
Good point, will rename.